File: TransCoder_tokenized_test_set_functions/python/FIND_MAXIMUM_ELEMENT_ROW_MATRIX.py
def maxelement ( arr ) :
    no_of_rows = len ( arr )
    no_of_column = len ( arr [ 0 ] )
    for i in range ( no_of_rows ) :
        max1 = 0
        for j in range ( no_of_column ) :
            if arr [ i ] [ j ] > max1 :
                max1 = arr [ i ] [ j ]
        print ( max1 )
    

File: TransCoder_tokenized_test_set_functions/cpp/FIND_MAXIMUM_ELEMENT_ROW_MATRIX.cpp
void maxelement ( int no_of_rows, int arr [ ] [ N ] ) {
  int i = 0;
  int max = 0;
  int result [ no_of_rows ];
  while ( i < no_of_rows ) {
    for ( int j = 0;
    j < N;
    j ++ ) {
      if ( arr [ i ] [ j ] > max ) {
        max = arr [ i ] [ j ];
      }
    }
    result [ i ] = max;
    max = 0;
    i ++;
  }
  printArray ( result, no_of_rows );
}